Modules/xSharePoint/DSCResources/MSFT_xSPManagedMetadataServiceApp/MSFT_xSPManagedMetaDataServiceApp.psm1

```diff
@@ -62,6 +64,10 @@ function Set-TargetResource
             
 
             if ($params.ContainsKey("InstallAccount")) { $params.Remove("InstallAccount") | Out-Null }
+            if ($params.ContainsKey("ContentTypeHubUrl")) {
+                $params.Add("HubUri", $params.ContentTypeHubUrl)
```

Comment:
Sorry, I lied I have another comment. We need a note added somewhere highlighting this value isn't currently updated after the initial provisioning